README.md

```diff
@@ -249,8 +249,11 @@ To improve Devnet performance, instead of calculating the actual hash of deploym
 Consider passing these CLI flags on Devnet startup:
 
 - `--lite-mode` enables all of the optimizations described below (same as using all of the flags below)
-- `--lite-mode-deploy-hash` disables the calculation of transaction hash for deploy transactions
-- `--lite-mode-block-hash` disables the calculation of block hash
+- `--lite-mode-deploy-hash`
+  - disables the calculation of transaction hash for deploy transactions
+- `--lite-mode-block-hash`
+  - disables the calculation of block hash
+  - disables get_state_update functionality
```

Comment:
Isn't this enabled now? `__update_state` returns the state_update and it's saved on the block?

Reply:
When you say "now", do you mean currently on master or with the changes introduced by this PR?

Reply:
Actually neither the behavior on master nor behavior introduced with this PR are good. What I meant when I wrote `disables get_state_update functionality` was that it doesn't work as expected. What would be better is to log a warning or return an error from the endpoint.

Reply:
Commit https://github.com/Shard-Labs/starknet-devnet/pull/105/commits/bf050eb33688921d0e59598f98964e9b272c50ae restores the old behavior of returning `None` in case of using lite mode. To make it clear that state updates aren't supported (in the sense that the response is not what the user would expect), it's either this (which results in state update being `null`) or returning an error.